oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/JdbcOAuth2AuthorizationService.java

```diff
@@ -164,17 +167,38 @@ public JdbcOAuth2AuthorizationService(JdbcOperations jdbcOperations,
 	 */
 	public JdbcOAuth2AuthorizationService(JdbcOperations jdbcOperations,
 			RegisteredClientRepository registeredClientRepository, LobHandler lobHandler) {
+		this(jdbcOperations, registeredClientRepository, lobHandler, new DefaultColumnTypeExtractor());
+	}
+
+
+
+	/**
+	 * Constructs a {@code JdbcOAuth2AuthorizationService} using the provided parameters.
+	 *
+	 * @param jdbcOperations             the JDBC operations
+	 * @param registeredClientRepository the registered client repository
+	 * @param lobHandler                 the handler for large binary fields and large text fields
+	 * @param columnTypeExtractor          the extractor of columns metadata
+	 */
+	public JdbcOAuth2AuthorizationService(JdbcOperations jdbcOperations,
```

Comment:
Please remove this constructor as we do not want to expose any Public API's for this change. Please ensure all changes are internal only. You will also need to ensure we don't expose `OAuth2AuthorizationRowMapper.setTokenValueColumnType()` and  `OAuth2AuthorizationParametersMapper.setTokenValueColumnType()`.